pkgs/development/tools/electron/generic.nix

```diff
@@ -39,17 +40,18 @@ let
   };
 
   headersFetcher = vers: hash: fetchurl {
-    url = "https://atom.io/download/electron/v${vers}/node-v${vers}-headers.tar.gz";
+    url = "https://artifacts.electronjs.org/headers/dist/v${vers}/node-v${vers}-headers.tar.gz";
     sha256 = hash;
   };
 
   tags = {
-    i686-linux = "linux-ia32";
     x86_64-linux = "linux-x64";
     armv7l-linux = "linux-armv7l";
     aarch64-linux = "linux-arm64";
     x86_64-darwin = "darwin-x64";
     aarch64-darwin = "darwin-arm64";
+  } // lib.optionalAttrs (lib.versionOlder version "19.0.0") {
```

Comment:
I think this is breaking eval because the attr is missing on 19.0.0 and get is returning unsupported platform which is not a valid hash. I think in line 58 a throw is missing to make this return valid syntax again.